db-utils/src/main/kotlin/com/icerockdev/common/database/sql/Table.kt

```diff
@@ -84,7 +79,7 @@ data class GeographyPoint(
     }
 }
 
-open class Json<out T : Any>(private val clazz: Class<T>, private val jsonMapper: ObjectMapper) : ColumnType() {
+open class Json<out T : Any>(private val typeReference: TypeReference<T>, private val jsonMapper: ObjectMapper) : ColumnType() {
```

Comment:
it's breaking change.
can we add secondary constructor with old args and map them automatically to new requirements?